### ISSUE TITLE
Not working at all for video urls from other origins

### ISSUE THREAD

**Issue by nightsurge** [yahoo/gifshot#48]
I wish I could give more details, but the code does not output any errors during my attempts and I did not have a chance to figure it out yet. Just reporting so I don't forget.

Basically whenever I switch the example project's "example.mp4" file for a video file hosted on a different origin, I never see a gif created. I see it create some video elements on the DOM, but nothing ever displays.

**Comment by NikhilSapra** [comment]
Thank you for that valuable info Mr. Josh. :) I'm new to github..like very noob..just started learning things.Your comments are useful to me.-Nikhil Sapra                                                                                                                                                                                                                                                                                                                                        Sent from my BlackBerry 10 smartphone.                                                                                                                                                                                                                From: Josh ArnoldSent: Monday 16 November 2015 19:40To: yahoo/gifshotReply To: yahoo/gifshotSubject: [gifshot] Not working at all for video urls from other origins (#48)I wish I could give more details, but the code does not output any errors during my attempts and I did not have a chance to figure it out yet. Just reporting so I don't forget.

Basically whenever I switch the example project's "example.mp4" file for a video file hosted on a different origin, I never see a gif created. I see it create some video elements on the DOM, but nothing ever displays.

—Reply to this email directly or view it on GitHub.

**Comment by gfranko** [comment]
Wherever you are pulling the video needs to allow CORS
